src/cashier_frontend_new/src/modules/auth/state/auth.svelte.ts

```diff
@@ -90,6 +92,51 @@ export const authState = {
   get pnp() {
     return pnp;
   },
+  // Get anonymous agent
+  get anonAgent() {
```

Comment:
Characters are for free, rename to `anonimousAgent()`. In addition, a getter is to return the value of a property; it is not appropriate to use it here.